server/gokbg3/src/main/groovy/org/gokb/CrossRefPkgRun.groovy

```diff
@@ -434,8 +449,8 @@ class CrossRefPkgRun {
           def he_result = titleHistoryService.processHistoryEvents(ti, titleObj, title_class_name, user, fullsync, locale)
           if (he_result.errors) {
             currentTippError.put(historyEvents: [message: messageService.resolveCode('crossRef.package.tipps.error.title.history', null, locale),
```

Comment:
Same issue as above, put expects two parameters. Also, since the historyEvents are only defined on a title level, the correct place in the hierarchy would be `title.historyEvents`